service/src/test/java/bio/terra/workspace/app/controller/FolderApiControllerTest.java

```diff
@@ -323,7 +323,8 @@ public void deleteFolders_deleteTopLevelFolder_folderAndSubFoldersAllDeleted() t
   private ApiJobReport deleteFolderAndWaitForJob(UUID workspaceId, ApiFolder folder)
       throws Exception {
     String serializedResponse =
-        deleteFolderExpectCode(workspaceId, folder.getId(), HttpStatus.SC_ACCEPTED)
+        deleteFolder(workspaceId, folder.getId())
+            .andExpect(status().is2xxSuccessful())
```

Comment:
Fix unrelated flakey unit test. If deleting the folder is fast enough, the api return 200, not 202 as the test expects. With this change it just cares about success.